internal/cache/base.go

```diff
@@ -0,0 +1,207 @@
+package cache
+
+import (
+	"os"
+	"sync"
+	"time"
+
+	"fyne.io/fyne/v2"
+)
+
+var (
+	cacheDuration     = 1 * time.Minute
+	cleanTaskInterval = cacheDuration / 2
+
+	canvasRefreshCh = make(chan struct{}, 1)
+	expiredObjects  = make([]fyne.CanvasObject, 0, 50)
```

Comment:
I made it global only to avoid having to recreate the slice on every clean task, in that way the same slice is reused on every task without the need to allocate memory again. Should I make it local?

Reply:
Making it global requires locking, surely being able to avoid that outweighs any time required in allocating memory?

Reply:
It does not require locking because cache.Clean won't run asynchronously

Reply:
Is this ok?

Reply:
@andydotxyz Any comment about this?